commands/list_organizations.go

```diff
@@ -55,30 +88,38 @@ func listOrgs(cmd *cobra.Command, args []string) {
 }
 
 // orgsTable fetches the organizations the user is a member of
-// and returns a table in string form
+// and returns a table in string form.
 func orgsTable() (string, error) {
+	auxParams := ClientV2.DefaultAuxiliaryParams()
+	auxParams.ActivityName = listOrgsActivityName
 
-	organizations, apiResponse, err := Client.GetUserOrganizations(listOrganizationsActivityName)
+	response, err := ClientV2.GetOrganizations(auxParams)
 	if err != nil {
-		return "", APIError{err.Error(), *apiResponse}
+		if clientErr, ok := err.(*clienterror.APIError); ok {
+			if clientErr.HTTPStatusCode == http.StatusUnauthorized {
+				return "", microerror.Mask(notAuthorizedError)
+			} else if clientErr.HTTPStatusCode == http.StatusForbidden {
+				return "", microerror.Mask(accessForbiddenError)
+			}
+		}
+
+		return "", microerror.Mask(err)
 	}
 
-	if apiResponse.Response.StatusCode == http.StatusOK {
-		var output string
-		if len(organizations) == 0 {
-			output = color.YellowString("No organizations available\n")
-		} else {
-			// sort orgs by Id
-			slice.Sort(organizations[:], func(i, j int) bool {
-				return organizations[i].Id < organizations[j].Id
-			})
-
-			output = color.CyanString("ORGANIZATION") + "\n"
-			for _, org := range organizations {
-				output = output + org.Id + "\n"
-			}
+	var output string
+	if len(response.Payload) == 0 {
+		output = color.YellowString("No organizations available\n")
+	} else {
+		// sort orgs by Id
+		slice.Sort(response.Payload[:], func(i, j int) bool {
+			return response.Payload[i].ID < response.Payload[j].ID
+		})
+
+		output = color.CyanString("ORGANIZATION") + "\n"
+		for _, org := range response.Payload {
+			output = output + org.ID + "\n"
```

Comment:
This looks like the table is rendered manually. Isn't there a lib we can use to do this?